cms/static/js/views/course_info_update.js

```diff
@@ -17,32 +23,53 @@ define(['js/views/validation', 'codemirror', 'js/models/course_update',
 
             initialize: function() {
                 this.template = this.loadTemplate('course_info_update');
-                this.render();
```

Comment:
That actually makes me wonder if simply not rendering in initialize is enough to solve the problem (without dealing with disabling the button).